app/lib/question_navigation.rb

```diff
@@ -163,4 +163,30 @@ class QuestionNavigation
     Questions::SuccessfullySubmittedController,
     Questions::FeedbackController,
   ].freeze
+
+  def self.determine_current_question(intake)
+    return nil if intake.completed_at?
+    return Questions::ConsentController.to_path_helper unless intake.primary_consented_to_service_at?
+
+    if intake.completed_yes_no_questions_at? && intake.document_types_definitely_needed.present?
+      return Documents::OverviewController.to_path_helper
+    end
+
+    # If yes/no questions completed + docs uploaded, start at InterviewSscheduling. Else, start at consent
+    i = intake.completed_yes_no_questions_at? ? FLOW.index(Questions::OverviewDocumentsController) : FLOW.index(Questions::ConsentController)
+    found_path = nil
+    while found_path.nil?
+      i += 1
+      next if i == FLOW.index(Questions::OverviewDocumentsController) # we handle documents seperately above
```

Comment:
"we handle documents seperately above" doesn't make much sense to me as an explanation